### ISSUE TITLE
Add Feature - Copy Network Printer UNC path context menu option

### ISSUE THREAD

**Issue by thedoctrine101** [clechasseur/pathcopycopy#36]
In Windows, in the Devices and Printers window, one should be able to right-click a network printer hosted by a print server and copy its UNC path in the same way as with network folders. Sometimes network printer names are complex, and very long and difficult to type, and are not always listed in Active Directory so it would be beneficial to have the option to right-click a network printer on someone else's computer, then paste it into an email, then have them send it over to you so that you can easily add it yourself if you do not have it. Even if the printer name would be easy to type, it would be still be beneficial to have that feature added.

**Comment by thedoctrine101** [comment]
Well perhaps WMI and/or the Registry can be used in this matter. Definitely feasable through the registry but I am sure its feasable through WMI as well, and if so, it is most likely much easier.

**Comment by thedoctrine101** [comment]
Did you try working on this one? I think that would be a great enhancement.

**Comment by clechasseur** [comment]
No, I did not check yet. I just checked and it's still possible to view network printers in the same manner as before on Windows 10, so perhaps this is feasible after all...

**Comment by clechasseur** [comment]
I no longer have access to network printers to test this. If anyone wants to take a stab at this one, I'd accept a PR. 😄

**Comment by thedoctrine101** [comment]
Here is a suggestion in the hopes that it could make this easier. When a printer is network based, you always have it in the printer's name, such as "PRINTERNAME on SERVERNAME", perhaps you can just take that name in string form that is either stored in WMI or in the Registry, and then manipulate that string so as to make \\SERVERNAME\PRINTERNAME
